garage/envs/mujoco/randomization/variation.py

```diff
@@ -1,7 +1,8 @@
+# flake8: noqa
```

Comment:
Why skip flake8 here as well as other files?

Reply:
I had to edit almost 100 files in order to changed the logger API. I couldn't add docstrings and fix formatting errors in every single one.

Reply:
Actually I think that specific file doesn't need it! I must have edited it and then undone my edit